sympy/utilities/iterables.py

```diff
@@ -477,7 +477,7 @@ def _interactive_traversal(expr, stage):
     return _interactive_traversal(expr, 0)
 
 
-def ibin(n, bits=0, str=False):
+def ibin(n, bits, str=False):
```

Comment:
When you delete the default argument you break all existing code which calls this function with one argument. This is called "changing the API" and this particular type of change would be considered a "regression" because what used to work would no longer work. It would be ok to make `bits=None` as on `digits`.